lib/twilio.js

```diff
@@ -0,0 +1,59 @@
+const twilio = require('twilio');
+const Promise = require('Bluebird');
```

Comment:
Node.js already supports Promise.reject and Promise.resolve. I don't think we need bluebird.